src/ocean/text/xml/Document.d

```diff
@@ -272,7 +272,24 @@ version(d)
 
         ***********************************************************************/
 
-        final Document header (Immut!(T)[] encoding = null)
+        final Document header ()
```

Comment:
Hmm this looks like a new feature, not a bug fix.

Reply:
Yeah this is an interesting one. It fixes the bug that you can't use the `header` function without allocation, but I chose to do it by adding a new method.

Reply:
Otherwise I would have introduced a breaking change by changing the API.